Programowanie Obiektowe/1.2Klasy/_1_3PolaKlasyZad2/Program.cs

```diff
@@ -0,0 +1,2 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Hello, World!");
```

Comment:
Czy to potrzebne?

Reply:
[b34ee15](https://github.com/broniuu/Praktyki2022/pull/2/commits/b34ee15e1e4d819d2d2042bd7d4a432f49b28db8)